packages/react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -725,8 +725,9 @@ class ContextualMenuInternal extends React.Component<IContextualMenuInternalProp
         };
         ariaLabellledby = id;
       } else {
-        headerContextualMenuItem = sectionProps.title;
-        ariaLabellledby = this._id + sectionProps.title.text?.replace(/\s/g, '');
+        const id = this._id + sectionProps.title.key.replace(/\s/g, '');
```

Comment:
One other thing I just realized (sorry), `id` should default to `sectionProps.title.id` if it's provided.
```suggestion
        const id = sectionProps.title.id || (this._id + sectionProps.title.key.replace(/\s/g, ''));
```

Reply:
Committed the suggestion